src/main/java/ca/corefacility/bioinformatics/irida/ria/web/ajax/metadata/dto/ProjectMetadataField.java

```diff
@@ -14,6 +13,9 @@ public class ProjectMetadataField {
 	private String type;
 	private String restriction;
 
+	public ProjectMetadataField() {
```

Comment:
It doesn't appear that we are using this constructor for this class anywhere. Is it required? If so, can you please add a comment for it?

Reply:
This is a DTO and the default constructor is required for serializing/deserializing the object. We have many DTOs in IRIDA with defautl constructors. Unless there's another way to do it? Please see dea1f2068e29394e60a6d257e8a6449ea2c08723.

Reply:
Awesome thanks for adding the comment to it!